Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM alpine:3.13
+ARG ARCH
+FROM ${ARCH}alpine:3.13
+
+ARG K0S_VERSION
 
 RUN apk add --no-cache bash coreutils findutils iptables curl tini
 
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.20.1/bin/linux/amd64/kubectl \
-       && chmod +x ./kubectl \
-       && mv ./kubectl /usr/local/bin/kubectl
+RUN curl -sSLf https://get.k0s.sh | K0S_VERSION=${K0S_VERSION} sh
```

Comment:
I don't think the binaries are downloadable at this point when we build the image 🤔

Reply:
I moved image building to the very last step of the GHA workflow, so I think it should work.

Reply:
Ok, now I see. I switched back to passing a binary via `ADD` in the Dockerfile